src/transformers/configureScope/transform.cjs

```diff
@@ -0,0 +1,182 @@
+      const scopeVarName = callbackFn.params[0]?.type === 'Identifier' ? callbackFn.params[0].name : undefined;
+
+      if (!scopeVarName) {
+        return;
+      }
+
+      const callbackBody = callbackFn.body;
+
+      if (callbackBody.type !== 'BlockStatement') {
+        return;
+      }
+
+      const getScopeMethodName = 'getCurrentScope';
+
+      if (path.value.callee)
+        if (callbackBody.body.length === 1 && callbackBody.body[0].type === 'ExpressionStatement') {
+          // If we only have a single statement inside, we can avoid the block
+          const statement = callbackBody.body[0];
+          // We check that we have a single statement that is e.g. scope.xxxx() only
+          if (
+            statement.expression.type === 'CallExpression' &&
+            statement.expression.callee.type === 'MemberExpression' &&
+            statement.expression.callee.object.type === 'Identifier' &&
+            statement.expression.callee.object.name === scopeVarName
+          ) {
+            const methodName = statement.expression.callee.property;
+
+            path.replace(
+              j.callExpression(
+                j.memberExpression(j.callExpression(j.identifier(getScopeMethodName), []), methodName),
+                statement.expression.arguments
+              )
+            );
+            return;
+          }
+        }
+
+      path.replace(
+        j.blockStatement([
+          j.variableDeclaration('const', [
+            j.variableDeclarator(j.identifier(scopeVarName), j.callExpression(j.identifier(getScopeMethodName), [])),
+          ]),
+          ...callbackBody.body,
+        ])
+      );
+    });
+
+    // Replace e.g. `SentryUtuils.configureScope()`
+    tree.find(j.CallExpression).forEach(path => {
+      if (
+        path.value.callee.type !== 'MemberExpression' ||
+        path.value.callee.property.type !== 'Identifier' ||
+        path.value.callee.property.name !== 'configureScope'
+      ) {
+        return;
+      }
+
+      const calleeObj = path.value.callee.object;
+
+      const callbackFn = path.value.arguments[0];
+      if (callbackFn.type !== 'ArrowFunctionExpression' && callbackFn.type !== 'FunctionExpression') {
+        return;
+      }
+
+      // The var name of the scope callback, e.g. (scope) => {} would be "scope"
+      const scopeVarName = callbackFn.params[0]?.type === 'Identifier' ? callbackFn.params[0].name : undefined;
+
+      if (!scopeVarName) {
+        return;
+      }
+
+      const callbackBody = callbackFn.body;
+
+      if (callbackBody.type !== 'BlockStatement') {
+        return;
+      }
+
+      // Very hacky, but we check if the callee (e.g. hub.configureScope() or getCurrentHub().configureScope())
+      // contains "hub", and if so, we use `getScope()` instead of `getCurrentScope()`
+      let getScopeMethodName = /hub/i.test(j(calleeObj).toSource()) ? 'getScope' : 'getCurrentScope';
```

Comment:
@Lms24 I added this somewhat hacky check, but I think it should work well enough...

Reply:
> there, it has to be `getScope()` instead of `getCurrentScope()` 😬 

Are we adding another transformer to replace `getCurrentHub().getScope()` with `getCurrentScope()`?

Reply:
Yeah, figured that too - we should also do that then! But I guess that will be a general `getCurrentHub()` transformer that rewrites that to all the new/global APIs instead!